admin/includes/functions/html_output.php

```diff
@@ -247,30 +247,45 @@ function zen_draw_form($name, $action, $parameters = '', $method = 'post', $para
     return $form;
   }
 
-////
-// Output a form input field
-  function zen_draw_input_field($name, $value = '~*~*#', $parameters = '', $required = false, $type = 'text', $reinsert_value = true) {
-    $type = zen_output_string($type);
-    if ($type === 'price') $type = 'number" step="0.01';
-
-    $field = '<input type="' . $type . '" name="' . zen_output_string($name) . '"';
-
-    if ( $value == '~*~*#' && (isset($GLOBALS[$name]) && is_string($GLOBALS[$name])) && ($reinsert_value == true) ) {
-      $field .= ' value="' . zen_output_string(stripslashes($GLOBALS[$name])) . '"';
-    } elseif ($value != '~*~*#' && zen_not_null($value)) {
-      $field .= ' value="' . zen_output_string($value) . '"';
-    }
-
-    if (zen_not_null($parameters)) $field .= ' ' . $parameters;
+  /**
+ * 
+ * Output a form input field
+ * @param string $name field name
+ * @param string $value field value
+ * @param string $parameters extra parameters like classes or id
+ * @param boolean $required field is required
+ * @param string $type filed type
+ * @param boolean $reinsert_value
+ * @return string
+ */
+function zen_draw_input_field($name, $value = '~*~*#', $parameters = '', $required = false, $type = 'text', $reinsert_value = true)
+{
+  $type = zen_output_string($type);
+  if ($type === 'price')
```

Comment:
Could you include braces around this clause?  Makes it more obvious that the single statement is to be executed for the condition.

Reply:
I left the original code alone, but I like to have braces as well. Will put them in.